packages/react-form/src/hooks/list/test/dynamiclist.test.tsx

```diff
@@ -73,6 +79,9 @@ describe('useDynamicList', () => {
     });
 
     it('will throw an error if new position is out of index range', () => {
+      const consoleErrorMock = jest.spyOn(console, 'error');
+      consoleErrorMock.mockImplementation();
```

Comment:
To avoid 
<img width="1193" alt="Screen Shot 2021-04-21 at 10 37 03 AM" src="https://user-images.githubusercontent.com/5626729/115605107-1ed2b000-a2b0-11eb-9c87-32c2f43e37d2.png">